assets/styles/style.css

```
@@ -0,0 +1,27 @@
+*{
+    box-sizing: border-box;
+}
+
+html,body{
```

It is better to start the second selector as a new line and not to forget adding a space between selector and open block like before.

 like so:
```suggestion
html,
body {
```